### PR TITLE
docs: Translate Exposing External IP Address Tutorial

### PR DESCRIPTION
**Pull request by irvifa**
Related to #22296
/cc @kubernetes/sig-docs-id-reviews

### CODE REVIEW

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+ yang membutuhkan sebuah penyedia layanan cloud.
+
+ * Konfigurasi `kubectl` agar dapat berkomunikasi dengan Kubernetes API Server kamu.
+ Untuk informasi lebih lanjut, kamu dapat merujuk pada dokumentasi penyedia layanan cloud
+ yang kamu gunakan.
+
+
+
+## {{% heading "objectives" %}}
+load-balancer-example.yaml
```
Comment (sftim) [inline content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:32]: Is this line right?

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+kubectl apply -f https://k8s.io/examples/service/load-balancer-example.yaml
+```
+
+
+Perintah di atas akan membuat sebuah
+    objek [Deployment](/docs/concepts/workloads/controllers/deployment/)
+    dan sebuah objek
+    [ReplicaSet](/docs/concepts/workloads/controllers/replicaset/)
+    yang diasosiasikan dengan deployment yang dibuat. ReplicaSet memiliki lima buah
+    [Pods](/docs/concepts/workloads/pods/pod/),
```
Comment (sftim) [inline content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:58]:
```suggestion
    objek [Deployment](/id/docs/concepts/workloads/controllers/deployment/)
    dan sebuah objek
    [ReplicaSet](/id/docs/concepts/workloads/controllers/replicaset/)
    yang diasosiasikan dengan deployment yang dibuat. ReplicaSet memiliki lima buah
    [Pods](/id/docs/concepts/workloads/pods/pod/),
```
?

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+
+    Jika sebuah alamat IP eksternal yang ditunjukkan dalam status \<pending\>, tunggulah hingga satu menit kemudian masukkan perintah yang sama lagi.
+
+    {{< /note >}}
+
+1. Tampilkan informasi detail mengenai Service:
+
+        kubectl describe services my-service
+
+    PErintah di atas akan menampilkan keluaran sebagai berikut:
```
Comment (sftim) [inline content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:100]:
```suggestion
    Perintah di atas akan menampilkan keluaran sebagai berikut:
```

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+ * Konfigurasi `kubectl` agar dapat berkomunikasi dengan Kubernetes API Server kamu.
+ Untuk informasi lebih lanjut, kamu dapat merujuk pada dokumentasi penyedia layanan cloud
+ yang kamu gunakan.
+
+
+
+## {{% heading "objectives" %}}
+
+* Jalankan lima buah instans dari aplikasi Hello World.
+* Buatlah sebuag objek Servis yang mengekspos sebuah alamat IP external.
```
Comment (wahyuoi) [inline content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:34]:
```suggestion
* Buatlah sebuah objek Service yang mengekspos sebuah alamat IP eksternal.
```

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+* Gunakan sebuah objek Servuce untuk mengakses aplikasi yang sedang dijalankan.
```
Comment (wahyuoi) [inline content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:35]:
```suggestion
* Gunakan sebuah objek Service untuk mengakses aplikasi yang sedang dijalankan.
```

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+
+
+
+<!-- lessoncontent -->
+
+## Membuat sebuah objek Servis untuk sebuah aplikasi yang dijalankan pada lima buah pod
```
Comment (wahyuoi) [inline content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:41]:
```suggestion
## Membuat sebuah objek Service untuk sebuah aplikasi yang dijalankan pada lima buah Pod
```

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md (hunk 2 above)
Comment (wahyuoi) [reply content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:57]: +1

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+
+1. Buatlah sebyah objek Service yang mengekspos deployment:
+
+        kubectl expose deployment hello-world --type=LoadBalancer --name=my-service
+
+1. Tampilkan informasi mengenai Service:
+
+        kubectl get services my-service
+
+    Keluaran dari Perintah di atas akan menyerupai tampilan berikut:
```
Comment (wahyuoi) [inline content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:78]:
```suggestion
    Keluaran dari perintah di atas akan menyerupai tampilan berikut:
```

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 8]
```diff
@@ -0,0 +1,172 @@
...
+
+        NAME         TYPE           CLUSTER-IP     EXTERNAL-IP      PORT(S)    AGE
+        my-service   LoadBalancer   10.3.245.137   104.198.205.71   8080/TCP   54s
+
+    {{< note >}}
+
+    Service `type=LoadBalancer` service didukung oleh penyedia layanan cloud external, yang tidak tercakup dalam contoh ini, silahkan merujuk pada [laman berikut](/docs/concepts/services-networking/service/#loadbalancer) untuk informasi lebih lanjut.
```
Comment (wahyuoi) [inline content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:85]:
```suggestion
    Servis dengan `type=LoadBalancer` didukung oleh penyedia layanan cloud eksternal, yang tidak tercakup dalam contoh ini, silahkan merujuk pada [laman berikut](/id/docs/concepts/services-networking/service/#loadbalancer) untuk informasi lebih lanjut.
```

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 9]
```diff
@@ -0,0 +1,172 @@
...
+
+    {{< /note >}}
+
+    {{< note >}}
+
+    Jika sebuah alamat IP eksternal yang ditunjukkan dalam status \<pending\>, tunggulah hingga satu menit kemudian masukkan Perintah yang sama lagi.
```
Comment (wahyuoi) [inline content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:91]:
```suggestion
    Jika sebuah alamat IP eksternal yang ditunjukkan dalam status \<pending\>, tunggulah hingga satu menit kemudian masukkan perintah yang sama lagi.
```

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 10]
```diff
@@ -0,0 +1,172 @@
...
+        Session Affinity:   None
+        Events:         <none>
+
+    Pastikan nilai dari alamat IP eksternal (`LoadBalancer Ingress`) diekspos
+    pada servis yang kamu buat. Pada contoh ini, alamat IP eksternal yang diberikan adalah 104.198.205.71.
+    Kemudian pastikan nilai dari `Port` dan `NodePort`. Pada contoh ini, `Port`
+    yang digunakan adalah 8080 dan `NodePort` adalah 32377.
+
+1. Pada keluaran Perintah sebelumnya, kamu dapat melihat beberapa service dengan beberapa endpoint:
+   10.0.0.6:8080,10.0.1.6:8080,10.0.1.7:8080 + 2 lainnya. Berikut ini merupakan alamat IP dari pod
```
Comment (wahyuoi) [inline content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:121]:
```suggestion
   10.0.0.6:8080,10.0.1.6:8080,10.0.1.7:8080 + 2 lainnya. Berikut ini merupakan alamat IP dari Pod
```

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 11]
```diff
@@ -0,0 +1,172 @@
...
+   dimana aplikasi tersebut dijalankan. Untuk melakukan verifikasi alamat-alamat IP yang digunakan oleh pod,
```
Comment (wahyuoi) [inline content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:122]:
```suggestion
   dimana aplikasi tersebut dijalankan. Untuk melakukan verifikasi alamat-alamat IP yang digunakan oleh Pod,
```

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 12]
```diff
@@ -0,0 +1,172 @@
...
+   masukkan Perintah berikut:
```
Comment (wahyuoi) [inline content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:123]:
```suggestion
   masukkan perintah berikut:
```

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 13]
```diff
@@ -0,0 +1,172 @@
...
+
+        kubectl get pods --output=wide
+
+    Keluarkan yang diberikan akan menyerupai:
```
Comment (wahyuoi) [inline content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:127]:
```suggestion
    Keluaran yang diberikan akan menyerupai:
```

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 14]
```diff
@@ -0,0 +1,172 @@
...
+        hello-world-2895499144-o4z13 ...  10.0.1.7   gke-cluster-1-default-pool-e0b8d269-1afc
+        hello-world-2895499144-segjf ...  10.0.2.5   gke-cluster-1-default-pool-e0b8d269-cpuc
+
+1. Gunakan alamat IP eksternal (`LoadBalancer Ingress`) untuk mengakses aplikasi Hello World:
+
+        curl http://<external-ip>:<port>
+
+    dimana `<external-ip>` adalah alamat IP eksternal (`LoadBalancer Ingress`)
+    dari Sevrice kamu, dan `<port>` adalah nilai dari `Port` dari deskripsi Service kamu.
+    Jika kamu menggunakan minikube, menuliskan Perintah `minikube service my-service` akan
```
Comment (wahyuoi) [inline content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:142]:
```suggestion
    Jika kamu menggunakan minikube, menuliskan perintah `minikube service my-service` akan
```

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 15]
```diff
@@ -0,0 +1,172 @@
...
+    secara otomatis membukan aplikasi Hello World di browser.
```
Comment (wahyuoi) [inline content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:143]:
```suggestion
    secara otomatis membukan aplikasi Hello World di _browser_.
```

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 16]
```diff
@@ -0,0 +1,172 @@
...
+
+    Response yang diberikan apabila request ini success adalah pesan hello:
```
Comment (wahyuoi) [inline content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:145]:
```suggestion
    Respons yang diberikan apabila permintaan ini berhasil adalah sebuah pesan sapaan:
```

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 17]
```diff
@@ -0,0 +1,172 @@
...
+
+        Hello Kubernetes!
+
+
+
+
+## {{% heading "cleanup" %}}
+
+
+Untuk menghapus Service, kamu dapat menggunakan Perintah ini:
```
Comment (wahyuoi) [inline content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:155]:
```suggestion
Untuk menghapus Service, kamu dapat menggunakan perintah ini:
```

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 18]
```diff
@@ -0,0 +1,172 @@
...
+
+    kubectl delete services my-service
+
+Untuk menghapus Deployment, ReplicaSet, dan Pod-Pod yang digunakan untuk
+menjalankan aplikasi Hello World, kamu dapat memasukkan Perintah berikut:
```
Comment (wahyuoi) [inline content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:160]:
```suggestion
menjalankan aplikasi Hello World, kamu dapat memasukkan perintah berikut:
```

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 19]
```diff
@@ -0,0 +1,172 @@
...
+    kubectl delete deployment hello-world
+
+
+
+
+## {{% heading "whatsnext" %}}
+
+
+Pelajari lebih lanjut cara untuk
+[menghubungkan aplikasi dengan servis-servis](/docs/concepts/services-networking/connect-applications-service/).
```
Comment (wahyuoi) [inline content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:171]:
```suggestion
[menghubungkan aplikasi dengan servis-servis](/id/docs/concepts/services-networking/connect-applications-service/).
```

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 20]
```diff
@@ -0,0 +1,172 @@
...
+        Endpoints:      10.0.0.6:8080,10.0.1.6:8080,10.0.1.7:8080 + 2 more...
+        Session Affinity:   None
+        Events:         <none>
+
+    Pastikan nilai dari alamat IP eksternal (`LoadBalancer Ingress`) diekspos
+    pada servis yang kamu buat. Pada contoh ini, alamat IP eksternal yang diberikan adalah 104.198.205.71.
+    Kemudian pastikan nilai dari `Port` dan `NodePort`. Pada contoh ini, `Port`
+    yang digunakan adalah 8080 dan `NodePort` adalah 32377.
+
+1. Pada keluaran Perintah sebelumnya, kamu dapat melihat beberapa service dengan beberapa endpoint:
```
Comment (wahyuoi) [inline content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:120]:
```suggestion
1. Pada keluaran perintah sebelumnya, kamu dapat melihat beberapa servis dengan beberapa endpoint:
```

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 21]
```diff
@@ -0,0 +1,172 @@
...
+```
+
+
+Perintah di atas akan membuat sebuah
+    objek [Deployment](/docs/concepts/workloads/controllers/deployment/)
+    dan sebuah objek
+    [ReplicaSet](/docs/concepts/workloads/controllers/replicaset/)
+    yang diasosiasikan dengan deployment yang dibuat. ReplicaSet memiliki lima buah
+    [Pods](/docs/concepts/workloads/pods/pod/),
+    yang masing-masing dari pod tersebut menjalankan aplikasi Hello World.
```
Comment (wahyuoi) [inline content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:58]:
```suggestion
    yang masing-masing dari Pod tersebut menjalankan aplikasi Hello World.
```

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 22]
```diff
@@ -0,0 +1,171 @@
...
+
+
+## {{% heading "prerequisites" %}}
+
+
+ * Instal [kubectl](/docs/tasks/tools/install-kubectl/).
+
+ * Gunakan sebuah penyedia layanan cloud seperti Google Kubernetes Engine atau Amazon Web Services
+ untuk membuat sebuah klaster Kubernetes. Tutorial ini membuat sebuah
+ [load balancer eksternal](/docs/tasks/access-application-cluster/create-external-load-balancer/),
```
Comment (girikuncoro) [inline content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:22]: this needs to refer to Indonesian page
also nit: italicize load balancer
```suggestion
 [_load balancer_ eksternal](/id/docs/tasks/access-application-cluster/create-external-load-balancer/),
```

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 23]
```diff
@@ -0,0 +1,171 @@
...
+Dokumen ini menjelaskan bagaimana cara membuat objek Service Kubernetes
+yang mengekspos alamat IP eksternal.
+
+
+
+
+## {{% heading "prerequisites" %}}
+
+
+ * Instal [kubectl](/docs/tasks/tools/install-kubectl/).
```
Comment (girikuncoro) [inline content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:18]: We already have the page for this, need to link to indonesian one
```suggestion
 * Instal [kubectl](/id/docs/tasks/tools/install-kubectl/).
```

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 24]
```diff
@@ -0,0 +1,171 @@
...
+```shell
+kubectl apply -f https://k8s.io/examples/service/load-balancer-example.yaml
+```
+
+
+Perintah di atas akan membuat sebuah
+    objek [Deployment](/id/docs/concepts/workloads/controllers/deployment/)
+    dan sebuah objek
+    [ReplicaSet](/id/docs/concepts/workloads/controllers/replicaset/)
+    yang diasosiasikan dengan deployment yang dibuat. ReplicaSet memiliki lima buah
```
Comment (girikuncoro) [inline content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:56]: nit
```suggestion
    yang diasosiasikan dengan Deployment yang dibuat. ReplicaSet memiliki lima buah
```

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 25]
```diff
@@ -0,0 +1,171 @@
...
+    [Pods](/id/docs/concepts/workloads/pods/pod/),
```
Comment (girikuncoro) [inline content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:57]: nit: singular is preferred
```suggestion
    [Pod](/id/docs/concepts/workloads/pods/pod/),
```

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 26]
```diff
@@ -0,0 +1,171 @@
...
+
+        kubectl get deployments hello-world
+        kubectl describe deployments hello-world
+
+1. Tampilkan informasi mengenai objek ReplicaSet:
+
+        kubectl get replicasets
+        kubectl describe replicasets
+
+1. Buatlah sebyah objek Service yang mengekspos deployment:
```
Comment (girikuncoro) [inline content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:70]: typo
```suggestion
1. Buatlah sebuah objek Service yang mengekspos deployment:
```

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 27]
```diff
@@ -0,0 +1,171 @@
...
+        kubectl get services my-service
+
+    Keluaran dari perintah di atas akan menyerupai tampilan berikut:
+
+        NAME         TYPE           CLUSTER-IP     EXTERNAL-IP      PORT(S)    AGE
+        my-service   LoadBalancer   10.3.245.137   104.198.205.71   8080/TCP   54s
+
+    {{< note >}}
+
+    Servis dengan `type=LoadBalancer` didukung oleh penyedia layanan cloud eksternal, yang tidak tercakup dalam contoh ini, silahkan merujuk pada [laman berikut](/id/docs/concepts/services-networking/service/#loadbalancer) untuk informasi lebih lanjut.
```
Comment (girikuncoro) [inline content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:85]: Service is kubernetes API object
```suggestion
    Service dengan `type=LoadBalancer` didukung oleh penyedia layanan cloud eksternal, yang tidak tercakup dalam contoh ini, silahkan merujuk pada [laman berikut](/id/docs/concepts/services-networking/service/#loadbalancer) untuk informasi lebih lanjut.
```

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 28]
```diff
@@ -0,0 +1,171 @@
...
+        IP:             10.3.245.137
+        LoadBalancer Ingress:   104.198.205.71
+        Port:           <unset> 8080/TCP
+        NodePort:       <unset> 32377/TCP
+        Endpoints:      10.0.0.6:8080,10.0.1.6:8080,10.0.1.7:8080 + 2 more...
+        Session Affinity:   None
+        Events:         <none>
+
+    Pastikan nilai dari alamat IP eksternal (`LoadBalancer Ingress`) diekspos
+    pada servis yang kamu buat. Pada contoh ini, alamat IP eksternal yang diberikan adalah 104.198.205.71.
```
Comment (girikuncoro) [inline content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:116]:
```suggestion
    pada Service yang kamu buat. Pada contoh ini, alamat IP eksternal yang diberikan adalah 104.198.205.71.
```

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 29]
```diff
@@ -0,0 +1,171 @@
...
+    Kemudian pastikan nilai dari `Port` dan `NodePort`. Pada contoh ini, `Port`
+    yang digunakan adalah 8080 dan `NodePort` adalah 32377.
+
+1. Pada keluaran perintah sebelumnya, kamu dapat melihat beberapa servis dengan beberapa endpoint:
```
Comment (girikuncoro) [inline content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:120]:
```suggestion
1. Pada keluaran perintah sebelumnya, kamu dapat melihat beberapa Service dengan beberapa endpoint:
```

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 30]
```diff
@@ -0,0 +1,171 @@
...
+        hello-world-2895499144-9m4h1 ...  10.0.0.6   gke-cluster-1-default-pool-e0b8d269-5v7a
+        hello-world-2895499144-o4z13 ...  10.0.1.7   gke-cluster-1-default-pool-e0b8d269-1afc
+        hello-world-2895499144-segjf ...  10.0.2.5   gke-cluster-1-default-pool-e0b8d269-cpuc
+
+1. Gunakan alamat IP eksternal (`LoadBalancer Ingress`) untuk mengakses aplikasi Hello World:
+
+        curl http://<external-ip>:<port>
+
+    dimana `<external-ip>` adalah alamat IP eksternal (`LoadBalancer Ingress`)
+    dari Sevrice kamu, dan `<port>` adalah nilai dari `Port` dari deskripsi Service kamu.
```
Comment (girikuncoro) [inline content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:141]: typo
```suggestion
    dari Service kamu, dan `<port>` adalah nilai dari `Port` dari deskripsi Service kamu.
```

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 31]
```diff
@@ -0,0 +1,171 @@
...
+    Jika kamu menggunakan minikube, menuliskan perintah `minikube service my-service` akan
+    secara otomatis membukan aplikasi Hello World di _browser_.
```
Comment (girikuncoro) [inline content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:143]: typo
```suggestion
    secara otomatis membuka aplikasi Hello World pada _browser_.
```

File: content/id/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 32]
```diff
@@ -0,0 +1,171 @@
...
+    kubectl delete deployment hello-world
+
+
+
+
+## {{% heading "whatsnext" %}}
+
+
+Pelajari lebih lanjut cara untuk
+[menghubungkan aplikasi dengan servis-servis](/id/docs/concepts/services-networking/connect-applications-service/).
```
Comment (girikuncoro) [inline content/id/docs/tutorials/stateless-application/expose-external-ip-address.md:171]: I think this refers to kubernetes API objects
```suggestion
[menghubungkan aplikasi dengan berbagai Service](/id/docs/concepts/services-networking/connect-applications-service/).
```

**Comment by girikuncoro** [comment]
/lgtm

need to fix the failed builds first

**Comment by irvifa** [comment]
@girikuncoro test is still failing? could you help me test on your local?

**Comment by irvifa** [comment]
@girikuncoro I rebased from master and it works now

**Comment by girikuncoro** [comment]
Very weird, anyway it works now, thanks @irvifa 

/lgtm
/approve

**Pull request merged**